tests/test_playbooks/auth_source_ldap.yml

```diff
@@ -42,11 +42,13 @@
   - include: tasks/auth_source_ldap.yml
     vars:
       auth_source_ldap_state: present
+      auth_source_ldap_account_password: "{{ omit }}"
       auth_source_ldap_netgroups: true
       expected_change: true
   - include: tasks/auth_source_ldap.yml
     vars:
       auth_source_ldap_state: present
+      auth_source_ldap_account_password: "{{ omit }}"
```

Comment:
Why not ` | default(omit)` in the tasks file?

Reply:
mostly because I was too lazy to touch the tasks file, but you're right.